src/users/users.controller.ts

```diff
@@ -16,5 +18,16 @@ export class UsersController {
     return this.userService.join(joinData);
   }
 
-  //TODO: POST login
+  @Post('login')
+  async login(@Res() response: Response, @Body() loginData: LoginDto): Promise<any> {
+    const jwt = await this.userService.login(loginData);
+
+    response.cookie('accessToken', jwt.accessToken, {
+      maxAge: 3600000,
```

Comment:
이런식으로 표현해주면 어떨까요? ms 단위이다 보니 큰 값이 그냥 적혀있으면 바로 이해하기가 어려울 것 같아요~
(주석은 취향일 것 같네요!)
```suggestion
      maxAge: 1000 * 60 * 60, // 1시간
```